R-package/demo/multiclass_custom_objective.R

```diff
@@ -92,17 +94,19 @@ custom_multiclass_metric <- function(preds, dtrain) {
     ))
 }
 
+params <- list(
+    min_data = 1L
+    , learning_rate = 1.0
+    , num_class = 3L
+)
 model_custom <- lgb.train(
     list()
```

Comment:
```suggestion
    params
```

Reply:
ah! Knew I'd miss one of these, thank you.